curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-subset-check-on-two-sets-of-data.english.md

```diff
@@ -21,17 +21,17 @@ For example, if <code>setA = ['a','b']</code> and <code>setB = ['a','b','c','d']
 ```yml
 tests:
   - text: Your <code>Set</code> class should have a <code>union</code> method.
-    testString: assert(function(){var test = new Set(); return (typeof test.subset === 'function')}, 'Your <code>Set</code> class should have a <code>union</code> method.');
+    testString: 'assert((function(){var test = new Set(); return (typeof test.subset === "function")})(), "Your <code>Set</code> class should have a <code>union</code> method.");'
   - text: The first Set() was contained in the second Set
-    testString: assert(function(){var setA = new Set(); var setB = new Set(); setA.add('a'); setB.add('b'); setB.add('c'); setB.add('a'); setB.add('d'); var subsetSetAB = setA.subset(setB);return (subsetSetAB === true)}, 'The first Set() was contained in the second Set');
+    testString: 'assert((function(){var setA = new Set(); var setB = new Set(); setA.add("a"); setB.add("b"); setB.add("c"); setB.add("a"); setB.add("d"); var subsetSetAB = setA.subset(setB);return (subsetSetAB === true)})(), "The first Set() was contained in the second Set");'
   - text: <code>['a', 'b'].subset(['a', 'b', 'c', 'd'])</code> should return <code>true</code>")
-    testString: assert(function(){var setA = new Set(); var setB = new Set(); setA.add('a'); setA.add('b'); setB.add('a'); setB.add('b'); setB.add('c'); setB.add('d'); var subsetSetAB = setA.subset(setB); return (subsetSetAB === true)}, "<code>['a', 'b'].subset(['a', 'b', 'c', 'd'])</code> should return <code>true</code>");
+    testString: 'assert((function(){var setA = new Set(); var setB = new Set(); setA.add("a"); setA.add("b"); setB.add("a"); setB.add("b"); setB.add("c"); setB.add("d"); var subsetSetAB = setA.subset(setB); return (subsetSetAB === true)})(), "<code>["a", "b"].subset(["a", "b", "c", "d"])</code> should return <code>true</code>");'
   - text: <code>['a', 'b', 'c'].subset(['a', 'b'])</code> should return <code>false</code>")
-    testString: assert(function(){var setA = new Set(); var setB = new Set(); setA.add('a'); setA.add('b'); setA.add('c'); setB.add('a'); setB.add('b'); var subsetSetAB = setA.subset(setB); return (subsetSetAB === false)}, "<code>['a', 'b', 'c'].subset(['a', 'b'])</code> should return <code>false</code>");
+    testString: 'assert((function(){var setA = new Set(); var setB = new Set(); setA.add("a"); setA.add("b"); setA.add("c"); setB.add("a"); setB.add("b"); var subsetSetAB = setA.subset(setB); return (subsetSetAB === false)})(), "<code>["a", "b", "c"].subset(["a", "b"])</code> should return <code>false</code>");'
```

Comment:
we need to either escape the double quotes or use single

Reply:
? Not sure but this is the file with errors, `missing ) after argument list`

Reply:
fixed missing bracket